src/platform/Darwin/PlatformManagerImpl.h

```diff
@@ -54,7 +55,7 @@ class PlatformManagerImpl final : public PlatformManager, public Internal::Gener
         return mWorkQueue;
     }
 
-    CHIP_ERROR PrepareCommissioning();
+    CHIP_ERROR PrepareCommissioning(PrepareCommissioningDelegate * delegate = nullptr);
```

Comment:
Again, how does one cancel this if the `PrepareCommissioningDelegate` is being destroyed?